net-misc/gallery-dl/gallery-dl-1.15.0.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=(python3_{7,8})
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit eutils distutils-r1 optfeature
+
+DESCRIPTION="Download image galleries and collections from several image hosting sites"
+HOMEPAGE="https://github.com/mikf/gallery-dl"
+
+if [[ "${PV}" == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/mikf/${PN}.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/mikf/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+RESTRICT="test"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="bash-completion zsh-completion"
```

Comment:
Are these necessary? https://devmanual.gentoo.org/general-concepts/use-flags/index.html

> USE flags must not be used to control installing files that are small, non-intrusive, do not introduce additional build-time dependencies or cause a significant increase in build time. Examples of such files are bash completion files

Reply:
In my opinion yes, because:
- many pkgs have those
- they are generated by https://github.com/mikf/gallery-dl/blob/master/Makefile#L42